config.go

```diff
@@ -32,8 +32,8 @@ func (c *Config) VaultPaths() []string {
 	}
 
 	return []string{
-		fmt.Sprintf("secret/global/%s/env_vars", c.Environment),             // DEPRECATED
-		fmt.Sprintf("secret/products/%s/env_vars", c.Product),               // DEPRECATED
+		fmt.Sprintf("secret/global/%s/%s/env_vars", c.Environment, c.Environment),             // DEPRECATED
```

Comment:
Is this correct? It's loading `secret/global/dev/dev/env_vars`?

Reply:
You're right, global was fine before, it was `products` that was broken.... updated.